### DISCUSSION TITLE
Flynt 2.0 Add Component Error

### DISCUSSION THREAD

**Discussion by purchasingtoolpak** [flyntwp/flynt#499] [Q&A]
I am using Flynt 2.0 with premium components. After I create a new page, then add the first component to the page - everything works fine. However if I try to add a second component - the add component button - does not enable the component drop down select menu. The button just appears frozen. I turned on debugging and I see no error messages.

Any debugging help would be appreciated.

Thanks

Dave

**Comment by timohubois** [comment]
@purchasingtoolpak tested it locally and can not reproduce the issue. Can you try to move folder of the component “FeatureFlexibleContentExtension” **temporary** to the theme main folder and then run `npm run build` and test it again?

**Reply by purchasingtoolpak** [reply]
Appreciate your response as it was very helpful.

I removed FeatureFlexibleContentExtension (FFCE) from the component folder. After run build and run start the AddComponent button did not freeze. I could add multiple components to any page. 
I put FFCE back into the component file then npm start - the component button was frozen. After removing the FFCE folder again the AddComponent button worked fine.
This is a good work around but I won't be able to use the FFCE component.

**Reply by timohubois** [reply]
@purchasingtoolpak did some tests again and can not reproduce the issue. Is the issue related to a specific browser?

**Comment by aaronmeder** [comment]
I've tried Flynt 2 locally and could replicate the issue (Firefox & Chrome). 

There seems to be a small mistake in the calculation of the "add component" tooltip, I'll prepare a fix.

Example positioning:
<img width="1709" alt="Bildschirmfoto 2023-08-04 um 10 28 11" src="https://github.com/flyntwp/flynt/assets/5564163/936601d3-9468-4f03-acdb-a46701dbcbd1">

**Comment by LuisSR** [comment]
Hi, @timohubois,

I also detected this a few days ago. I even captured a screencast of the issue:

https://github.com/flyntwp/flynt/assets/24281689/0a24c06c-0747-4dc4-92ec-87ac22aebeb3

It's a simple positioning problem, as @aaronmeder says. I didn't dig any deeper on what's causing it and have not tried Aaron fix yet.

Tested using Firefox 115.0.2 (the browser in the video) and Opera 101.0.4843.25 on Kubuntu 23.04.
